sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/AvoidExcessiveInheritance.cs

```diff
@@ -103,6 +106,15 @@ protected override void Initialize(ParameterLoadingAnalysisContext context)
                 }, SyntaxKind.ClassDeclaration);
         }
 
+        private static string GetRootNamespace(INamedTypeSymbol namedSymbol)

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Consider using more general type 'Microsoft.CodeAnalysis.ISymbol' instead of 'Microsoft.CodeAnalysis.INamedTypeSymbol'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS3242)